libs/designsystem/icon/src/kirby-icon-settings.ts

```diff
@@ -1,8 +1,13 @@
-import { IconSettings } from './icon-settings';
+import { Icon, IconSettings } from './icon-settings';
+
+const sortIconsByName = (icons: Icon[]) => {
+  const collator = new Intl.Collator();
+  return icons.sort((a, b) => collator.compare(a.name, b.name));
```

Comment:
Wondering about the use of `Intl.Collator`: all our icons are named in English and we don't construct the collator with any locales, so I guess it always runs the comparison in English anyhoo? 🤔

Consider simplifiying to `icons.sort()`

Reply:
I don't think we can use just the `.sort()` without providing a compare function, because `icons` is an array of *objects*, not numeric values.

At least we need to specify which properties the `sort()` method should compare, e.g.:

```
.sort((a, b) => a.name - b.name))
```

but that doesn't seem to work.

The best/safest way to sort an array of objects by a string property I could find is to use `localeCompare()` 

```
.sort((a, b) => a.name.localeCompare(b.name))
```

MDN states that 

> In implementations with [Intl.Collator API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator) support, this method simply calls Intl.Collator

We can change it to use `localeCompare()` since that must be equivalent to the current implementation using `Intl.Collator`

- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare

Reply:
Another option is to use the "sort by name" example from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#sorting_array_of_objects

It's more verbose, but quite straight-forward